lib/checking_account.rb

```diff
@@ -0,0 +1,75 @@
+require_relative './account'
+require 'csv'
+
+module Bank
+  class MoneyMarketAccount < Bank::Account
```

Comment:
Why is this `MoneyMarketAccount` - I would have expected to see `CheckingAccount` in this file (and that's what you appear to have implemented).